board/app/helpers/html_helper.php

```diff
@@ -10,4 +10,62 @@ function redirect($url)
 	header("Location: " . $url);
 	exit();
 }
-?>
+
+function time_difference($date)
```

Comment:
Done po. Thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/0cdfa167653f56ce5637ca4c4b4ac4b905c7e888